### PR TITLE
Spin up production pods in Skyline [push image]

### PR DESCRIPTION
**Pull request by pinkertonpg**
## What
Remove `skip_production_deploy`.

Remove a couple of other unused service.yaml lines.

## Why
Let's get this bad boy into production.

### CODE REVIEW

File: service.yml [hunk 1]
```diff
@@ -1,8 +1,5 @@
 name: avro-schema-registry
-image: avro-schema-registry
```
Comment (wpeterson) [inline service.yml]: Is this inferred from the `name` value above?

File: service.yml [hunk 2]
```diff
@@ -1,8 +1,5 @@
...
 slack: pb-s-sparkle-motion
-skip_production_deploy: true
-dockerfile: Dockerfile
```
Comment (wpeterson) [inline service.yml]: Is this now defaulted to `Dockerfile`?

File: service.yml (hunk 1 above)
Comment (pinkertonpg) [reply service.yml]: Yep 👍 

And the registry is always our production registry (as of v1.1 at least... previously it was staging when run against staging kube and prod for prod kube).

File: service.yml (hunk 2 above)
Comment (pinkertonpg) [reply service.yml]: I'm not sure this was ever used, to be honest. This was one of the earliest repos with a service.yaml. Way back in late March we added the initial service.yaml, this line included: https://github.com/ezcater/avro-schema-registry/pull/18

**Pull request merged**